sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,122 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
+  <student-page-container :layout-template="LayoutTemplates.Centered">
+    <template #header>
+      <header-navigator
+        title="Applications"
+        subTitle="Start New Application"
+        :routeLocation="{
+          name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+        }"
+      >
+      </header-navigator>
     </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
+    <v-form ref="startApplicationForm">
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for new funding
+      </h2>
+      <content-group>
+        <h3 class="category-header-medium-small pa-2">
```

Comment:
mm this is not an  heading in Figma, not sure if I am  missing anything 
![image](https://user-images.githubusercontent.com/77353155/199856390-b31ea8bc-c833-43e2-980e-53c7e3eb31d7.png)

Reply:
Please see https://github.com/bcgov/SIMS/pull/1527#discussion_r1013495800 Thanks!

Reply:
okay..looks like in the other Figma design also its  not a heading, maybe u can check with @hellolynn-tbtb 
![image](https://user-images.githubusercontent.com/77353155/199857289-050e9197-fac1-429b-bab8-316dec6722a6.png)

Reply:
PLease see https://github.com/bcgov/SIMS/pull/1527#discussion_r1013176207

Reply:
@ann-aot, Lynn fixed that on figma and I changed accordingly. Thanks!

Reply:
Changed from h3 to p according to conversation with Lynn. thanks!